lib/ansible/plugins/action/win_reboot.py

```diff
@@ -29,9 +30,10 @@ class ActionModule(RebootActionModule, ActionBase):
         'reboot_timeout', 'reboot_timeout_sec', 'shutdown_timeout', 'shutdown_timeout_sec', 'test_command',
     ))
 
+    DEFAULT_BOOT_TIME_COMMAND = "(Get-WmiObject -ClassName Win32_OperatingSystem).LastBootUpTime"
     DEFAULT_CONNECT_TIMEOUT = 5
     DEFAULT_PRE_REBOOT_DELAY = 2
-    DEFAULT_BOOT_TIME_COMMAND = "(Get-WmiObject -ClassName Win32_OperatingSystem).LastBootUpTime"
+    DEFAULT_POST_REBOOT_DELAY = 0
```

Comment:
Since this is the same value as defined in `reboot.py` do we need it here?

Reply:
Probably not, but I liked it to be clear locally.

And the other discussion we are having with the Windows WG is whether we should increase this to something more safe, we know that a post reboot delay of 0 is akin to troubles.

We definitely want to add a note to the documentation (in another PR) with respect to the general issue on Windows (not knowing when it has settled after a reboot) and add some examples of good validation checks that people can reuse.

Reply:
That shouldn't hold up this PR, we should file a new issue around adding more examples and improving the documentation for win_reboot and follow it up once this bug is fixed.

Reply:
Doc update is here: https://github.com/ansible/ansible/pull/46377